docs/capi-aws.md

```diff
@@ -0,0 +1,177 @@
+# Cluster API - AWS
+
+In this guide, we show you how to use AWS for the worker plane with the k0smotron control plane.
+
+## Prerequisites
+
+We assume that a k0s cluster has already been installed and is ready for use. 
+The kubeconfig should be available in ~/.kube/config and the context should already be set to the k0s cluster. 
+
+Next to the cluster we will need the following tools installed on your system:
+
+1. clusterctl CLI
+    - https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl
+2. clusterawsadm CLI
+    - https://github.com/kubernetes-sigs/cluster-api-provider-aws#clusterawsadm
+
+Also, you need to have an AWS account and your AWS CLI configured with your credentials.
+
+### Prepare the infra provider
+
+Prior to initiating a cluster, the configuration of the infrastructure provider is necessary. Each provider comes with its own unique set of prerequisites.
+
+The AWS infrastructure provider requires the `clusterawsadm` to be installed:
+``` bash
+curl -L https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v0.0.0/clusterawsadm-darwin-amd64 -o clusterawsadm
+chmod +x clusterawsadm
+sudo mv clusterawsadm /usr/local/bin
+```
+
+`clusterawsadm` is CLI which will prepare your AWS account, so it can be used as ClusterAPI Infrastructure Provider. 
+
+To begin, create the environment variables that specify the AWS account to be utilized, in case they have not been previously defined:
+``` bash
+export AWS_REGION=<your-region-eg-us-east-1>
+export AWS_ACCESS_KEY_ID=<your-access-key>
+export AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
+```
+
+In case you are using multi-factor authentication, you will need:
+
+``` bash
+export AWS_SESSION_TOKEN=<session-token> 
+```
+
+`clusterawsadm` will use these details to create a CloudFormation stack in your AWS account with IAM resources:
+
+``` bash
+clusterawsadm bootstrap iam create-cloudformation-stack
+```
+
+Ensure that the credentials are encoded and securely stored as a Kubernetes secret:
+
+``` bash
+export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile)
+```
+
+### Initialize the management cluster
+
+The initialization of Cluster API components requires the `clusterctl` to be installed:
```

Comment:
I would also remove the installation instructions, again we don't want to maintain this and also only works for mac with m chips.